sample/src/main/assets/json.form/diagnosis_form.json

```diff
@@ -63,53 +63,46 @@
         "openmrs_entity": "",
         "openmrs_entity_id": "",
         "buttonText": "+ Add disease code",
+        "sort": true,
         "dialogTitle": "Add disease code",
         "searchHint": "Type Disease Name",
-        "value": "[{\"key\":\"Bacterial Meningitis\",\"property\":{\"presumed-id\":\"er\",\"confirmed-id\":\"er\"}}]",
+        "value": "[{\"key\":\"bacterial_meningitis\",\"text\":\"Bacterial Meningitis\",\"property\":{\"presumed-id\":\"er\",\"confirmed-id\":\"er\"}}]",
```

Comment:
Do you need to save the whole selected object? Can't we just save the `keys` just like the `Radio Button` & `Native Forms` widget.